packages/babel-node/test/.babelrc

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
I know it's odd that it worked before. Maybe there's an underlying problem? The test throws this error when there's no `.babelrc`:

```sh
Error: Unsupported preset format: object. Expected preset to return a function. 
(While processing preset: "/Users/chitchu/Documents/Repo/babel/node_modules/babel-preset-es2015/lib/index.js")
```

The same thing happens with the original babel-node located in babel-cli.